libs/design-system/content-templates/google-maps/src/lib/google-maps.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, computed, inject, input } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+import { HraCommonModule } from '@hra-ui/common';
+import { ConsentService } from '@hra-ui/common/analytics';
+import { EventCategory } from '@hra-ui/common/analytics/events';
+import { ButtonsModule } from '@hra-ui/design-system/buttons';
+import { PrivacyPreferencesService } from '@hra-ui/design-system/privacy';
+
+/**
+ * Google Maps wrapper component to load and display Google Maps.
+ */
+@Component({
+  selector: 'hra-google-maps',
+  imports: [HraCommonModule, ButtonsModule],
+  templateUrl: './google-maps.component.html',
+  styleUrl: './google-maps.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+})
+export class GoogleMapsComponent {
+  /** DOM sanitizer */
+  private readonly sanitizer = inject(DomSanitizer);
+
+  /** Maps URL for the iframe */
+  readonly mapsUrl = input.required({
```

Comment:
Rename to `url`. Also please don't use `transform` for sanitizing the url. Use a separate `computed` property instead